terraform/stacks/external/stack.tf

```diff
@@ -1,5 +1,5 @@
 module "cdn_broker" {
```

Comment:
And if we move the bucket creation to `modules/s3_bucket/cdn_broker_bucket` how does this module definition change? can we have multiple source? Would there be a `cdn_broker_bucket` module in this file now instead of just a single `module "cdn_broker"`?